src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -20,6 +20,10 @@
 
   <PropertyGroup>
     <RuntimeIdentifier>$(__DistroRid)</RuntimeIdentifier>
+
+    <!-- TODO-LLVM: how should this work, building subset clr.tools to get a `build` folder, DistroRid is browser-wasm, but we want the host Rid -->
+    <RuntimeIdentifier Condition="'$(TargetArchitecture)' == 'wasm'">$(AppHostRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
I don't think we want to build `clr.tools` for WASM, only the host.

If we need `build` in the WASM artifacts, we should resurrect `nativeaot.build` as a separate subset and use that.